Dockerfile

```
@@ -0,0 +1,8 @@
+FROM gradle:8-jdk17 AS build
+WORKDIR /app
+COPY . .
+RUN gradle bootJar --no-daemon
+
+FROM eclipse-temurin:17-jre
+COPY --from=build /app/build/libs/*.jar app.jar
```

Hadolint (2.12.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)